cvat/schema.yml

```diff
@@ -6152,6 +6771,53 @@ components:
         * `annotation` - ANNOTATION
         * `validation` - VALIDATION
         * `completed` - COMPLETED
+    JobType:
+      enum:
+      - annotation
+      - ground_truth
+      type: string
+      description: |-
+        * `annotation` - ANNOTATION
+        * `ground_truth` - GROUND_TRUTH
+    JobWriteRequest:
+      type: object
+      properties:
+        assignee:
+          type: integer
+          nullable: true
+        stage:
+          $ref: '#/components/schemas/JobStage'
+        state:
+          $ref: '#/components/schemas/OperationStatus'
+        type:
+          $ref: '#/components/schemas/JobType'
+        task_id:
+          type: integer
+        frame_selection_method:
+          $ref: '#/components/schemas/FrameSelectionMethodEnum'
+        count:
```

Comment:
This should probably be called `frame_count`, or it's a bit unclear.

Reply:
Updated